### DISCUSSION TITLE
PostgreSQL Security

### DISCUSSION THREAD

**Discussion by EthanKC** [nextgenhealthcare/connect#5212] [Q&A] [answered]
I'm just starting to dip my toe into PostgreSQL. I work for a company where I'm installing/configuring/maintaining Mirth on several different client servers. We are a passthrough, so the only info that will be stored on the database is what Mirth stores there. I found these instructions, which I used to install PostgreSQL on my own device, and set it up with my own instance of Mirth as practice: https://github.com/rbeckman-nextgen/test-mc/wiki/Setting-Up-Mirth-Using-PostgreSQL

It seems to be working, but those instructions specifically say they don't cover security: 

_Warning: This method doesn't cover database security. If this is a public facing server then please seek out help on proper PSQL security practices._

I think I'm probably good, because none of the servers I'm installing on should be public facing. But I wanted to do my due diligence, since there will be patient health information processing through these databases. So, is there anything else I should be aware of/try to implement security wise?

**Comment by ab-20-m** [comment, marked as answer]
I would review the [How to Secure Postgresql: Security Hardening Best Practices & Tips](https://www.enterprisedb.com/blog/how-to-secure-postgresql-security-hardening-best-practices-checklist-tips-encryption-authentication-vulnerabilities) for a starting point on securing the Postgres instance(s) themselves.
